assets/levels/level_bindings.gd

```diff
@@ -5,8 +5,6 @@ extends Node3D
 @export var game_cam: Camera3D
 
 func _ready():
-	Savegame.load_file()
```

Comment:
Moved this into the GameManager so it loads before the rest of the scene initialises